src/components/request/RequestForm.tsx

```diff
@@ -269,29 +269,19 @@ const RequestForm: FC = () => {
             {(flags.CanDispute || flags.InDvmVote) && (
               <ProposerAddress>
                 Proposer:{" "}
-                <a
-                  target="_blank"
-                  rel="noreferrer"
-                  href={`${explorerUrl}/address/${proposer}`}
-                >
-                  {proposer}
+                <a target="_blank" rel="noreferrer" href={exploreProposeTx}>
+                  {proposeTx}
```

Comment:
this can be really long, but its not clear what the best way to deal with it is, added a string shortener in case we do need something like that